test.js

```diff
@@ -1,18 +1,14 @@
-'use strict';
-var test = require('ava');
-var npmUser = require('./');
+import test from 'ava';
+import fn from './';
 
-test(function (t) {
-	t.plan(7);
+test(async t => {
+	const user = await fn('sindresorhus');
 
-	npmUser('sindresorhus', function (err, user) {
-		console.log(user);
-		t.assert(!err, err);
-		t.assert(user.name === 'Sindre Sorhus');
-		t.assert(user.email === 'sindresorhus@gmail.com');
-		t.assert(user.homepage === 'http://sindresorhus.com');
-		t.assert(user.github === 'sindresorhus');
-		t.assert(user.twitter === 'sindresorhus');
-		t.assert(user.freenode === 'sindresorhus');
-	});
+	console.log(user);
```

Comment:
Previously, the assert methods didn't show what output if it didn't match, so just logged it as a temp workaround.